diff --git a/node_modules/metro-resolver/src/resolve.js b/node_modules/metro-resolver/src/resolve.js
index b533ed1..0078b75 100644
--- a/node_modules/metro-resolver/src/resolve.js
+++ b/node_modules/metro-resolver/src/resolve.js
@@ -479,6 +479,9 @@ function resolveWindowsPath(modulePath) {
 }
 
 function isRelativeImport(filePath) {
+  if (filePath.startsWith('..\\')) {
+    return true;
+  }
   return /^[.][.]?(?:[/]|$)/.test(filePath);
 }
 
